unicornia/src/Blocks/custom/news/news.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Template for the News Block view.
+ *
+ * @package Unicornia
+ */
+
+use UnicorniaVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = $attributes['blockClass'] ?? '';
+
+$blockTitle = Components::checkAttr('content', $attributes, $manifest);
+
+$spaceNews = new \Unicornia\SpaceNews\SpaceNews();
+$news = $spaceNews->fetch();
+?>
+
+<div class="<?php echo esc_attr($blockClass); ?>">
+	<?php echo \wp_kses_post(Components::render('heading', [
+		'headingContent' => $blockTitle ?? ''
+	])); ?>
+	<div class="<?php echo esc_attr("{$blockClass}__content-wrap {$blockClass}__content-wrap--default"); ?> block-news-js">
+
+		<?php foreach ($news as $newsItem) {
+			?>
+			<div class="<?php echo esc_attr("{$blockClass}__post"); ?>"
+				 data-extid="<?php echo esc_attr($newsItem['id']); ?>">
+				<?php
+				echo \wp_kses_post(Components::render('image', ['imageUrl' => $newsItem['imageUrl']]));
+				?>
+				<span class="<?php echo esc_attr("{$blockClass}__post__meta"); ?>">
+					<i><?php echo \wp_kses_post(gmdate("d.m.Y @ H:i", strtotime($newsItem['publishedAt']))); ?></i>
+				</span>
+				<?php
+				echo \wp_kses_post(Components::render('heading', ['headingContent' => $newsItem['title']]));
+				?>
+				<?php
+				echo \wp_kses_post(Components::render('paragraph', ['paragraphContent' => $newsItem['summary']]));
+				?>
+				<div class="<?php echo esc_attr("{$blockClass}__post__tags"); ?>">
```

Comment:
See comment above, BEM classes should have only 1 element.

```suggestion
				<div class="<?php echo esc_attr("{$blockClass}__post-tags"); ?>">
```